Mathlib/CategoryTheory/Conj.lean

```diff
@@ -118,4 +106,34 @@ theorem map_conjAut (F : C ⥤ D) {X Y : C} (α : X ≅ Y) (f : Aut X) :
 -- alternative proof: by simp only [Iso.conjAut_apply, F.mapIso_trans, F.mapIso_symm]
 end Functor
 
+namespace Equivalence
+
+universe uC uC' vC vC' uD uD' vD vD'
+
+variable {C : Type uC} [Category.{vC} C]
+variable {C' : Type uC'} [Category.{vC'} C']
+variable {D : Type uD} [Category.{vD} D]
+variable {D' : Type uD'} [Category.{vD'} D']
+variable {f : C ⥤ D} {g : C' ⥤ D'}
+variable {e : C ≌ C'} {e' : D ≌ D'}
+
+/--
+Suppose we have categories `C, C', D, D'` such that the diagram of functors
+```
+C ===== f =====> D
+||e            ||e'
+||             ||
+C' ==== g ====> D'
+```
+commutes where `e` and `e'` are equivalence of categories.
```

Comment:
It doesn't really make sense to say a diagram of functors commutes, because a category is only defined up to equivalence. Maybe "commutes up to natural isomorphism" or something?

Reply:
done